### PR TITLE
Use batches to call CreateTimeSeries

### PR DESCRIPTION
**Pull request by knightpp**
Based on https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/575 I've updated sending logic to batch time series data. The argument is hardcoded, and I couldn't find mocks for `*monitoring.MetricClient` so no tests.

### CODE REVIEW

**Comment by knightpp** [comment]
> just need to you to sign the CLA (see the bot comment)
Hi, I've signed the CLA. There's a checkmark on the PR. Thanks.

**Comment by damemi** [comment]
Ah sorry, I see it now.
/gcbrun

File: exporter/metric/metric.go [hunk 1]
```diff
@@ -223,29 +223,48 @@ func (me *metricExporter) createMetricDescriptorIfNeeded(ctx context.Context, md
...
 	}
 
-	// TODO: When this exporter is rewritten, support writing to multiple
-	// projects based on the "gcp.project.id" resource.
-	req := &monitoringpb.CreateTimeSeriesRequest{
-		Name:       fmt.Sprintf("projects/%s", me.o.projectID),
-		TimeSeries: tss,
+	name := fmt.Sprintf("projects/%s", me.o.projectID)
+
+	const batchSize = 200
```
Comment (damemi) [inline exporter/metric/metric.go:246]: could you define this at the top-level of the file with a comment explaining it, like in the collector exporter https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/e9bb597e4aad734e5fa68ac8847e7000303368de/exporter/collector/metrics.go#L104-L108

**Review by damemi — COMMENTED** [review]
looks good, one nit and can you add a test too? The test might be easier if you make `batchSize` a configurable (private) field on the exporter, eg so you can run a test with a max batch size of like 1 or 2 instead of having to create 200+ test metrics

**Comment by knightpp** [comment]
> can you add a test too? 

Added tests and refactored `exportTimeSeries`.

**Pull request merged**